docs/en/latest/getting-started.md

```diff
@@ -21,247 +21,264 @@ title: Getting Started
+It will take some time to download all needed files, and this depends on your network, please be patient. Once this step gets done, we could `curl` our Admin API to tell if the Apache APISIX launchs successfully.
 
 ```bash
+# NOTE: Please curl on the machine which you run above Docker commands.
 $ curl "http://127.0.0.1:9080/apisix/admin/services/" -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1'
 ```
 
-The following will be the response from the Admin API.
+We expect the following data to be returned:
 
 ```json
 {
-    "node": {
-        "createdIndex": 6,
-        "modifiedIndex": 6,
-        "key": "/apisix/services",
-        "dir": true
-        },
-    "action": "get"
+  "node": {
+    "createdIndex": 6,
+    "modifiedIndex": 6,
+    "key": "/apisix/services",
+    "dir": true
+  },
+  "action": "get"
 }
 ```
 
-## Step 2: Create a Route in APISIX
-
-APISIX provides a powerful Admin API and a [dashboard](https://github.com/apache/apisix-dashboard) for configuring the routes/services/plugins.
-The quickstart guide will use the Admin API for configuring the routes.
+## Step 2: Create a Route
 
-A micro-service can be configured via APISIX through the relationship between several entities such as routes, services, upstream, and plugins.
-The route matches the client request and specifies how they are sent to the upstream (backend API/Service) after they reach APISIX.
-Services provide an abstraction to the upstream services. Therefore, you can create a single service and reference it in multiple routes.
-Check out the architecture document for more information.
+Congratulations! You have a running Apache APISIX instance now! Let's create a Route next!
 
-Technically all this information(upstream or service, plugins) can be included inside a route configuration. The route consists of three main parts.
+### Before we continue
 
-- Matching Rules:
+Do you know? Apache APISIX provides the powerful [Admin API](./admin-api.md) and a [Dashboard](https://github.com/apache/apisix-dashboard) for us to use, but we will use Admin API here in this guide. Let's go!
 
-    Let's take the following scenario.
-    http://example.com/services/users
+We could create one [Route](./architecture-design/route.md) and target it to our backend services (We call them [Upstream](./architecture-design/upstream.md) usually), when one `Request` reaches Apache APISIX, Apache APISIX will see where this Request should go.
 
-    The URL above hosts all the micro services related to the users(getUser/ GetAllUsers) in the system. For example the GetAllUsers endpoint can be reached via the following URL (http://example.com/services/users/GetAllUsers)
-    Now you want to expose all the `GET` endpoints(micro-services) under the `users` path. The following will be the route configuration for matching such request.
+Then how does Apache APISIX know this? that's because we have a list of rules configured with Route. Here is an sample Route data:
```

Comment:
an sample => an example of?

Reply:
Hi, I just updated a typo: an sample -> a sample;

I would prefer using `a sample xxx` than `an example of xxx` 🤔 the first one is more frankly speaking IMO.